airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2109,3 +2128,265 @@ def execute(self, context: Any):
+                    'store_backend': {
+                        'class_name': 'TupleGCSStoreBackend',
+                        'project': self.gcp_project,
+                        'bucket': self.gcs_bucket,
+                        'prefix': self.gcs_validations_prefix
+                    }
+                },
+                "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+            },
+            data_docs_sites={
+                "GCS_site": {
+                    "class_name": "SiteBuilder",
+                    "store_backend": {
+                        "class_name": "TupleGCSStoreBackend",
+                        "project": self.gcp_project,
+                        "bucket": self.gcs_bucket,
+                        'prefix': self.gcs_datadocs_prefix
+                    },
+                    "site_index_builder": {
+                        "class_name": "DefaultSiteIndexBuilder",
+                    },
+                }
+            },
+            config_variables_file_path=None,
+            commented_map=None,
+        )
+        data_context = BaseDataContext(project_config=project_config)
+
+        # Tell GE how to fetch the batch of data that should be validated.
+        batch_kwargs = {
+            "datasource": "bq_datasource",
+        }
+        if self.validation_type == great_expectations_valid_type.SQL.name:
+            batch_kwargs["query"] = self.validation_type_input
+            batch_kwargs["data_asset_name"] = self.bq_dataset_name
+            batch_kwargs["bigquery_temp_table"] = self.get_temp_table_name()
+        elif self.validation_type == great_expectations_valid_type.TABLE.name:
+            batch_kwargs["table"] = self.validation_type_input
+            batch_kwargs["data_asset_name"] = self.bq_dataset_name
+
+        log.info("batch_kwargs: " + str(batch_kwargs))
+
+        log.info("Loading expectations...")
+        suite = data_context.get_expectation_suite((self.expectations_file_name.rsplit(".", 1)[0]))
+
+        log.info("Getting the batch of data to be validated...")
+        batch = data_context.get_batch(batch_kwargs, suite)
+
+        run_id = {
+            "run_name": 'bq',
+            "run_time": datetime.datetime.now(datetime.timezone.utc)
+        }
+
+        log.info("Validating batch against expectations...")
+        results = data_context.run_validation_operator(
+            "action_list_operator",
+            assets_to_validate=[batch],
+            run_id=run_id)
+
+        validation_result_identifier = list(results['run_results'].keys())[0]
+        # For the given validation_result_identifier, get a link to the data docs that were generated by Great
+        # Expectations as part of the validation.
+        data_docs_url = \
+            data_context.get_docs_sites_urls(resource_identifier=validation_result_identifier, site_name='GCS_site')[0][
+                'site_url']
+        log.info("Data docs url is: " + data_docs_url)
```

Comment:
```suggestion
        log.info("Data docs url is: %s", data_docs_url)
```
To be pylint compatible